dev/pages/Attention.vue

```diff
@@ -97,7 +97,6 @@ const highlightShowing = ref(false)
             <w-box
               neutral
               as="h4"
-              :ref="calloutTarget ? calloutTarget.$el : null"
```

Comment:
I'm guessing that we should remove the declaration on line 7 as well then? 😄 
I'm also a bit curious about whether the other similar refs and targets actually do anything? 🤔 🤷

Reply:
Right! I will remove the declaration on line 7. Hmm not sure. We don't need the ref for the interactive elements? 🙈 🤔

Reply:
Not related to this line but I looked at line 61 and am curious - shouldn't ref value on the target element be just `ref="tooltipResetTarget"`? I thought this way we are assigning that element reference to the `tooltipResetTarget` constant, which then can be passed to Attention's `target-el` prop.

Reply:
Whoops you’re right! I’ll replace it with ”tooltipResetTarget” :)